nebula-logger/main/log-management/classes/LogEntryHandler.cls

```diff
@@ -39,6 +42,9 @@ public without sharing class LogEntryHandler {
                 this.setCheckboxFields();
             }
         }
+
+        // Run any post-processors
+        super.executePostProcessors(this.logEntries, this.oldLogEntriesById);
```

Comment:
I would avoid using `super` here because you're not overriding the `executePostProcessors` method - `this.executePostProcessors(...)` will suffice!

Reply:
Yeah, I couldn't decide if using `super` or `this` was more intuitive - I'll switch to `this` instead!